_posts/2022-04-19-arrow-r-cheatsheet.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: post
+title: Arrow for R Cheatsheet
+date: "2022-04-19 00:00:00"
+author: stephhazlitt
+categories: [application]
+---
+<!--
+{% comment %}
+Licensed to the Apache Software Foundation (ASF) under one or more
+contributor license agreements.  See the NOTICE file distributed with
+this work for additional information regarding copyright ownership.
+The ASF licenses this file to you under the Apache License, Version 2.0
+(the "License"); you may not use this file except in compliance with
+the License.  You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+{% endcomment %}
+-->
+
+We are excited to introduce the new [Arrow for R Cheatsheet](https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow-cheatsheet.pdf).
+
+<div align="center">
+<a href="https://github.com/apache/arrow/blob/master/r/cheatsheet/arrow-cheatsheet.pdf">
+<img src="{{ site.baseurl }}/img/20220419-arrow-r-cheatsheet-thumbnail.png"
+     alt="Thumbnail image of the first page of the Arrow for R cheatsheet."
+     width="70%" height="70%">
+</a>
+</div>
+
+## Helping (Not Cheating)
+
+While [cheatsheets](https://en.wikipedia.org/wiki/Cheat_sheet) may have started as a set of notes used without an instructor’s knowledge&mdash;so, ummm, cheating&mdash;using the Arrow for R cheatsheet is definitely not cheating! Today, cheatsheets are a common tool to provide users an introduction to software’s functionality and a quick reference guide to help users get started.
+
+The Arrow for R cheatsheet is intended to be an easy-to-scan introduction to the Arrow R package and Arrow data structures, with getting started sections on some of the package’s main functionality. The cheatsheet includes introductory snippets on using Arrow to read and work with larger-than-memory multi-file data sets, sending and receiving data with Flight, reading data from cloud storage without downloading the data first, and more. The Arrow for R cheatsheet also directs users to the full [Arrow for R package documentation and articles](https://arrow.apache.org/docs/r/index.html) and the [Arrow Cookbook](https://arrow.apache.org/cookbook/), both full of code recipes to support users build their Arrow-based data workflows. Finally, the cheatsheet debuts one of the first uses of the hot-off-the-presses Arrow hex sticker, recently made available as part of the [Apache Arrow visual identity guidance](https://arrow.apache.org/visual_identity/).
```

Comment:
```suggestion
The Arrow for R cheatsheet is intended to be an easy-to-scan introduction to the Arrow R package and Arrow data structures, with getting started sections on some of the package’s main functionality. The cheatsheet includes introductory snippets on using Arrow to read and work with larger-than-memory multi-file data sets, sending and receiving data with Flight, reading data from cloud storage without downloading the data first, and more. The Arrow for R cheatsheet also directs users to the full [Arrow for R package documentation and articles](https://arrow.apache.org/docs/r/index.html) and the [Arrow Cookbook](https://arrow.apache.org/cookbook/r/), both full of code recipes to support users build their Arrow-based data workflows. Finally, the cheatsheet debuts one of the first uses of the hot-off-the-presses Arrow hex sticker, recently made available as part of the [Apache Arrow visual identity guidance](https://arrow.apache.org/visual_identity/).
```

Perhaps consider changing "both full of code recipes" to "both full of code examples and recipes" as the pedant in me says that it's only the cookbook which contains recipes.